src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/java14/InputJava14InstanceofWithPatternMatching.java

```diff
@@ -0,0 +1,75 @@
+package com.puppycrawl.tools.checkstyle.grammar.java14;
```

Comment:
There should be `non-compiled` marker. See
https://github.com/checkstyle/checkstyle/blob/master/src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/java9/InputJava9TryWithResources.java#L1 for reference

Reply:
Fixed.